### DISCUSSION TITLE
Maintain Notion Sites menu navigation

### DISCUSSION THREAD

**Discussion by ajcoach** [velsa/notehost#40] [Ideas]
Hi velsa,
since Notion now has their own Sites feature it would be great if notehost could maintain the existing menu bar from a Notion Site, effectively having the same capability as Notion Sites but hosted on your own domain for free.

**Comment by DudeThatsErin** [comment]
What do you mean? Can you screenshot what you are referring to?

**Reply by DiegoDerksen** [reply]
I'm pretty sure that he means the following navbar: 
![image](https://github.com/user-attachments/assets/57c6345c-e0a5-48a5-9a91-5cd90869b2c7)

I got it back after deleting the (non working) dark more/light mode switch. I'll try to look if I can get the dark/light mode to work somehow.. But I'm kinda glad that I also got this navbar back.
